src/app/games/lodestone/lodestone.component.html

```diff
@@ -20,11 +20,11 @@
                    class="lodestone_crumbled_square">
                     <polygon points="0,0     42,0    17,25    67,75    42,100   0,100   0,0"
                              [ngClass]="getSquareClasses(x, y)"
-                             class="base"
+                             class="base crumbled-square-left"
```

Comment:
data !!!!!!

Reply:
En vrai, just put real ids !!!!

Reply:
ça commence ici à être limite, genre un niveau d'id en plus et ça serait ridicule, on se retrouvera à faire "#player-zero-lodestone-little-triangle-on-the-left-decoration-on-the-triangle", plutôt que de faire juste la description du chemin en sélecteur :p